core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/PluginResolver.scala

```diff
@@ -5,16 +5,13 @@ import org.virtuslab.ideprobe.dependencies.Plugin._
 import org.virtuslab.ideprobe.error
 
 object PluginResolver {
-  // TODO (#253): replace Official with a default value in reference.conf - `probe.resolvers.plugins.repository.uri`
-  val Official: DependencyResolver[Plugin] = PluginResolver("https://plugins.jetbrains.com/plugin/download")
```

Comment:
Again, please do not drop useful apis. Read it from config if necessary, but keep the val.

Reply:
sure, will be fixed in upcomming commit